### DISCUSSION TITLE
Tree 란 , 종류

### DISCUSSION THREAD

**Discussion by starshb** [starshb/pharmacy_Find#46] [Q&A] [answered]
Tree 란 , 종류

**Comment by tjdud22** [comment, marked as answer]
트리(Tree)는 힙(Heap)을 구현하는 방법 중 하나로 그래프의 일종으로 정점과 간선을 이용하여 데이터의 배치 형태를 추상화한 자료구조이다.
서로 다른 두 노드를 연결하는 길이 하나뿐인 그래프를 트리라고 부른다.
트리 자료구조는 일반적으로 대상 정보의 각 항목들을 계층적으로 구조화할 때 사용하는 비선형 자료구조이다.

종류

삼항 트리 (Ternary tree) : 최대 세개의 자식을 같은 구조
이진트리 (Binary Tree) : 최대 두 개의 자식을 같은 구조
